pkg-manager/core/package.json

```diff
@@ -81,17 +81,16 @@
     "@pnpm/store-path": "workspace:*",
     "@pnpm/test-fixtures": "workspace:*",
     "@types/fs-extra": "^9.0.13",
-    "@types/is-ci": "^3.0.0",
     "@types/is-windows": "^1.0.0",
     "@types/normalize-path": "^3.0.0",
     "@types/ramda": "0.28.20",
     "@types/semver": "7.3.13",
     "@types/sinon": "^10.0.13",
     "@yarnpkg/core": "4.0.0-rc.27",
+    "ci-info": "3.7.0",
```

Comment:
Are there any rules on when to use exact vs patch ranges for deps? Seeing that it is somehow mixed...

Reply:
External package are mostly using `^`. Exact versions are for pnpm dependencies and for some exceptions.

Reply:
I see, thank you for the clarification! I've updated the PR accordingly.